ush/parsing_namelists_CICE.sh

```diff
@@ -80,8 +80,6 @@ cat > ice_in <<eof
    restart_FY   = .false.
    tr_lvl       = .true.
    restart_lvl  = .false.
-   tr_pond_cesm = .false.
```

Comment:
can you add a comment in the PR description, so this does not seem like trying to sneak in?

Reply:
Done